mara_app/monkey_patch.py

```diff
@@ -41,6 +50,13 @@ def patch(original_function: typing.Callable) -> typing.Callable:
 
     Returns: The replaced function
     """
+    logger.warning(
+        f'function {original_function.__module__}.{original_function.__name__} is being replaced (change: {patch_description})')
+    __applied_patches.append(Patch(replaces=True,
```

Comment:
This is basically a registry.

This only works if you import `monkey_patch` from this packages. But that would mean you depend on `mara_app` which the downloaders should not. So you would end up copying the file to your package and you will end up with different version if this file :-( -> I think this should go to a different package (`mara_config`) which all other packages should depend on and implement a registry there.